freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/upscale/action/FreeIpaUpscaleActions.java

```diff
@@ -179,7 +180,8 @@ protected void doExecute(StackContext context, StackEvent payload, Map<Object, O
                 if (newInstances.isEmpty()) {
                     skipAddingNewInstances(context, stack);
                 } else {
-                    addNewInstances(context, stack, newInstances);
+                    List<String> instanceIdsToRemove = getInstanceIds(variables);
```

Comment:
this could be potentially `null` if the flow(chain) is started with the previous version of FMS, but interrupted with the release and continued on this version. The `addNewInstances` method expects this to be non null